Fitness/Models/ProRoutine.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  RoutineType.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 3/2/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import Foundation
+
+struct ProRoutine {
+    let title: String
+    let routineType: ProRoutineType
+    let text: String
+    
+    init(title: String, routineType: ProRoutineType, text: String) {
+        self.title = title
+        self.routineType = routineType
+        self.text = text
+    }
+}
+
+enum ProRoutineType {
```

Comment:
same comment here

Reply:
I'll move those two to be within their respective classes